README.md

```diff
@@ -263,27 +231,24 @@ Note:
 You can view your change with `systemctl cat grub-btrfsd`.
 To revert all the changes use `systemctl revert grub-btrfsd`.
 
-##### ❇️ Automatically update grub upon restart/boot:
-[Look at this comment](https://github.com/Antynea/grub-btrfs/issues/138#issuecomment-766918328)  
-Currently not implemented
 - - -
-#### OpenRC instructions
-To start the daemon run
+#### grub-btrfsd OpenRC instructions
+To start the daemon run:
 ```bash
 sudo rc-service grub-btrfsd start 
 ```
 
-To activate it during system startup, run 
+To activate it during system startup, run:
 ```bash
 sudo rc-config add grub-btrfsd default 
 ```
 
-##### 💼 Snapshots not in `/.snapshots`
+##### 💼 Snapshots not in `/.snapshots` for OpenRC
 NOTE: This works also for Timeshift versions < 22.06, the path to watch would be `/run/timeshift/backup/timeshift-btrfs/snapshots`.
```

Comment:
I know you haven't added this, but let's get rid of this. It is not true in most circumstances anymore and for it we have the flag `--timeshift-old`. The timeshift folder is only mounted when timeshift opens, so the daemon would fail when it is started before. Since it can not find the folder. 
The only case when this would work is if people put the  `/run/timeshift/back....` folder into fstab.